Workbooks/Virtual Machines/Virtual Machine Performance/Single Virtual Machine Performance.workbook

```diff
@@ -337,7 +336,7 @@
       "type": 3,
       "content": {
         "version": "KqlItem/1.0",
-        "query": "let metric = dynamic({Counter});\r\nPerf\r\n| where TimeGenerated {TimeRange}\r\n| where Computer in ({VirtualMachines}) or '*' in ({VirtualMachines})\r\n| where Computer == '{computer}' or 'All' == '{computer}'\r\n| where ObjectName == metric.object and CounterName == metric.counter\r\n| summarize {Metrics} by bin(TimeGenerated, totimespan('{grain}'))",
```

Comment:
are you removing the All option?

Reply:
Yes, that clause is not necessary since only a single VM can be displayed at a time